Automattic/wp-calypso
# Fix: import upload redirect

#### Changes proposed in this Pull Request

1. Fix: After reloading a page with an uploading pane element and you have already uploaded a file, an importing pane element must be shown. An importer status `UPLOAD_SUCCESS` must redirect you to importing pane
2. Move the `analyzeUrl` call from the `WordPress` flow to parent `ImportOnboardingFrom`. This is done because all the flows use it and so a `UrlData` is updated everywhere

#### Testing instructions

* Go directly to `http://calypso.localhost:3000/start/from/importing/IMPORTER_TYPE?from=IMPORT_FROM&to=IMPORT_TO`
* Check if `https://public-api.wordpress.com/wpcom/v2/imports/analyze-url?_envelope=1&site_url=IMPORT_FROM` is called (proposed change **2**)

If you are testing an upload flow like Medium etc.
* Import the export file
* Reload the page
* Check if the import pane is shown instead of the upload pane (proposed change **1**)

Related to #60495
